packages/suite/src/support/messages.ts

```diff
@@ -5996,6 +5996,11 @@ export default defineMessages({
         defaultMessage: 'These outputs are likely smaller than the fee required to spend them.',
         description: 'Sub-heading in Coin control section',
     },
+    TR_SEARCH_UTXOS: {
+        id: 'TR_SEARCH_UTXOS',
+        defaultMessage: 'Search transactions',
```

Comment:
I think it shall be something like: `Search UTXOs`

cc @Hermez-cz for better copy